config.yml

```diff
@@ -405,6 +405,9 @@ nodes:
     child_nodes:
       - name: arguments
         type: node[]
+      - name: keywords
+        type: node?
+        kind: KeywordHashNode
```

Comment:
As a note, it could just be `HashNode` and we could remove `KeywordHashNode`, because it's implied by this field.
OTOH KeywordHashNode doesn't have opening_loc/closing_loc so KeywordHashNode is better footprint-wise.